cri/Makefile

```diff
@@ -40,6 +40,25 @@ test:
 
 	go test ./... $(EXTRAGOARGS)
 
+test-gvisor-cri:
```

Comment:
does it run locally?

Reply:
Yes, tested on xl170. Did not work for you?